### PR TITLE
test(frontend): Derived store tokens list

### PR DESCRIPTION
**Pull request by AntonioVentilii**
# Motivation

Some tests for the derived store `tokens`.

### CODE REVIEW

File: src/frontend/src/tests/lib/derived/tokens.derived.spec.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		ledgerCanisterId: `${mockValidIcToken.ledgerCanisterId}2`,
+		version: 1n,
+		enabled: true
+	};
+
+	describe('tokens', () => {
+		beforeEach(() => {
+			vi.resetAllMocks();
+
+			erc20DefaultTokensStore.reset();
```
Comment (peterpeterparker) [inline src/frontend/src/tests/lib/derived/tokens.derived.spec.ts:55]: In the Gix channel d.K. recently shared a snippet to reset all stores of the app for test purpose if I remember correctly. Maybe it's something we can reuse? (for another PR)

File: src/frontend/src/tests/mocks/erc20-tokens.mock.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { ETHEREUM_NETWORK } from '$env/networks.env';
+import type { Erc20Token } from '$eth/types/erc20';
+import { parseTokenId } from '$lib/validation/token.validation';
+import { mockEthAddress } from '$tests/mocks/eth.mocks';
+import { mockValidToken } from '$tests/mocks/tokens.mock';
+
+export const mockValidErc20Token: Erc20Token = {
```
Comment (peterpeterparker) [inline src/frontend/src/tests/mocks/erc20-tokens.mock.ts:7]: Standard "erc20" maybe?

**Review by peterpeterparker — APPROVED** [review]
Thx!

File: src/frontend/src/tests/mocks/erc20-tokens.mock.ts (hunk 2 above)
Comment (AntonioVentilii) [reply src/frontend/src/tests/mocks/erc20-tokens.mock.ts:7]: better! done

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/tests/lib/derived/tokens.derived.spec.ts (hunk 1 above)
Comment (AntonioVentilii) [reply src/frontend/src/tests/lib/derived/tokens.derived.spec.ts:55]: Done with:

- https://github.com/dfinity/oisy-wallet/pull/4429

**Pull request merged**